### PR TITLE
Idr0030 CMPO term additions

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/fFjB4Uh2/4-idr0030-sero-yap

### CODE REVIEW

File: scripts/reannotate/prod45_idr0030_input.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Screen:1801 idr0030-sero-yap/screenA openmicroscopy.org/mapr/phenotype
```
Comment (eleanorwilliams) [inline scripts/reannotate/prod45_idr0030_input.txt]: This needs to be `Screen:1801 idr0030-sero-yap/screenA/idr0030-screenA openmicroscopy.org/mapr/phenotype`

File: scripts/reannotate/prod45_idr0030_input_bulk.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Screen:1801 idr0030-sero-yap/screenA
```
Comment (eleanorwilliams) [inline scripts/reannotate/prod45_idr0030_input_bulk.txt]: This needs to be `Screen:1801 idr0030-sero-yap/screenA/idr0030-screenA`

**Comment by sbesson** [comment]
Started to testing the re-annotation of this study on `test45` by running:

```
$ /opt/omero/server/OMERO.server/bin/omero login demo@localhost
$ /opt/omero/server/OMERO.server/bin/omero delete OriginalFile:15107105
$ git clone https://github.com/snoopycrimecop/idr-metadata
$ cd idr-metadata
$ ./scripts/reannotate/bulk.sh scripts/reannotate/prod45_idr0030_input_bulk.txt
$ ./scripts/reannotate/delete.sh scripts/reannotate/prod45_idr0030_input.txt
$ ./scripts/reannotate/annotate.sh scripts/reannotate/prod45_idr0030_input.txt
```

Unfortunately, the first execution of `delete.sh` failed with a timeout similar to the one reported in https://trello.com/c/kRPgBR36/30-linked-annotation-deletion-is-very-slow-due-to-53-graph-deletion-rules. Restarted the deletion in a screen.

**Comment by sbesson** [comment]
After a few retries of the `delete.sh` script, the annotation deletion completed successfully and so did the reannotation. Ready for review on the testing server.

**Comment by eleanorwilliams** [comment]
Checked in idr-testing (test45).  The new ontology mappings are there both through wells and through the mapr phenotype category, and links go to the correct terms in CMPO via OLS.  Also checked  a few wells against production idr to check nothing has changed. 

All looks good.   We could add a note in the study file to say these have been added but for 2 only it might not be worth it.

**Pull request merged**